library/js/oeUI/headerTitleAction.js

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Expand Contract State and Show /Hide State js
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+?>
+$(document).ready(function () {
+    $('.expand_contract').click(function () {
+        var elementTitle = $(this).prop('title');
+        var expandTitle = '<?php echo xlj('Click to Contract and set to henceforth open in Centered mode'); ?>';
+        var contractTitle = '<?php echo xlj('Click to Expand and set to henceforth open in Expanded mode'); ?>';
+        var arrFiles = <?php echo json_encode($arr_files_php) ?>;
+
+        if ($(this).is('.oe-expand')) {
+            elementTitle = expandTitle;
+            $(this).toggleClass('fa-expand fa-compress');
+            $(this).toggleClass('oe-expand oe-center');
+            $('#container_div').toggleClass('container container-fluid');
+            if ($(arrFiles).length) {
+                $.each(arrFiles, function (index, value) {
+                
+                    $.post(
+                        "<?php echo $GLOBALS['webroot'] ?>/library/ajax/user_settings.php",
+                        {
+                            target: arrFiles[index].trim(),
+                            setting: 1,
+                            csrf_token_form: "<?php echo attr(collectCsrfToken()); ?>"
```

Comment:
here's was yo ensure proper js escaping
```
csrf_token_form: <?php echo js_escape(collectCsrfToken()); ?>
```
(note the end quotes were removed)

Reply:
Tx